homeassistant/components/zwave.py

```diff
@@ -165,6 +170,10 @@ def start_zwave(event):
         """ Called when Home Assistant starts up. """
         NETWORK.start()
 
+        polling_interval = config[DOMAIN].get(CONF_POLLING_INTERVAL, None)
+        if polling_interval is not None:
+            NETWORK.setPollInterval(polling_interval)
```

Comment:
Zwave does this per node basis. As far as I understand, each node has a poll "intensity" which can be 0, 1 or 2. 0 means never poll. 1 means once per interval and 2 means every other interval (did not quite get it). Now in the docs of OpenZwave it says the user should adjust the interval to the amount of polled devices, so that the network would not be jammed by all those poll requests (generally a value of 1000ms per polled node is recommended)